src/Test2.test.js

```diff
@@ -0,0 +1,66 @@
+/**
+ * @jest-environment jsdom
+ */
+import editTask from './modules/editTask.js';
+import taskStatusUpdate from './modules/taskStatusUpdate.js';
+import list from './modules/list.js';
+import clearCompletedTasks from './modules/clearCompletedTasks.js';
+
+describe('Edit task', () => {
+  let tasks = [];
+  const newTask = {
+    description: 'New task',
+    completed: false,
+    id: tasks.length + 1,
+  };
+
+  test('Clear completed test', () => {
+    document.body.innerHTML = `
+            <ul class="to-do-list">
+            <li id="to-do-title" class="list-item">Today's To Do</li>
+            <li class="list-item">
+                <input class="input" type="text" placeholder="Add to your list...">
+            </li>
+            </ul>
+        <a class="clear">Clear all completed</a>
+        `;
+
+    // pushing new tasks into the array
+    let i = 0;
+    while (i < 8) {
+      const newTask = { description: `Task ${i}`, completed: i % 2 !== 0 };
+      tasks.push(newTask);
+      i += 1;
+    }
+
+    const container = document.querySelector('.to-do-list');
+    tasks.forEach((task) => {
+      list(container, task, '#');
+    });
```

Comment:
- [ ] **[OPTIONAL]** Since here you're arranging your `tasks` list and displaying it to the DOM, I would suggest using the `beforeAll` hook, which basically runs before all your tests. This way, you keep inside the tests only what you're testing and avoid including this setup. 🤓 🔧